src/Tribe/Admin/Settings.php

```diff
@@ -38,5 +38,34 @@ public function should_load_image_field_assets() {
         }
         return $load_assets;
     }
+
+    /**
+     * Load color field assets if not already loaded.
+     * 
+     * @since TBD
+     *
+     * @return void
+     */
+    public function maybe_load_color_field_assets() {
+        if ( has_filter( 'tribe_admin_load_color_field_assets', '__return_true' ) ) {
+            return;
+        }
+        add_filter( 'tribe_admin_load_color_field_assets', '__return_true' );
+    }
+
+    /**
+     * Logic to load color field assets.
+     * 
+     * @since TBD
+     *
+     * @return bool
+     */
+    public function should_load_color_field_assets() {
+        $load_assets = apply_filters( 'tribe_admin_load_color_field_assets', false );
```

Comment:
Filters should use `tec_` prefix. I would love to see this make mention that this is related to the setting page and docblocks.